Python/GiraCave1.py

```diff
@@ -65,7 +65,7 @@ def __init__(self):
         self.version = 1
 
     def OnServerInitComplete(self):
-        if type(sdlSolved.value) == type("") and sdlSolved.value != "":
+        if isinstance(sdlSolved.value, str) and sdlSolved.value != "":
```

Comment:
`if sdlSolved.value:` - `ptAttribString.value` is always set to an str by the engine.